commands/export.go

```diff
@@ -82,6 +82,7 @@ func init() {
 	ExportCreateCmd.Flags().Bool("attachments", false, "Set to true to include file attachments in the export file.")
 
 	ExportDownloadCmd.Flags().Bool("resume", false, "Set to true to resume an export download.")
+	ExportDownloadCmd.Flags().Int("num_retries", 5, "Number of retries to do to resume a download.")
```

Comment:
We use dashes for flags :)
```suggestion
	ExportDownloadCmd.Flags().Int("num-retries", 5, "Number of retries to do to resume a download.")
```

Reply:
Ah I was thinking that we were migrating from dashes to underscores. Thanks!